packages/cypress/cypress.config.js

```diff
@@ -6,7 +6,7 @@ const { createEsbuildPlugin } = require('@badeball/cypress-cucumber-preprocessor
 
 module.exports = defineConfig({
   video: true,
-  defaultCommandTimeout: 20000,
+  defaultCommandTimeout: 40000,
```

Comment:
@dkayiwa @jayasanka-sack I think 40 seconds ain't so low and not too high at the same time. 
Most times cypress timesout while trying to find the patient search button, but works when you reload the test.  I think the home page takes alittle long to load all components 

However, this only occurs while running the tests locally.